install/install_requirements.sh

```diff
@@ -47,7 +47,7 @@ fi
 # NOTE: If a newly-fetched version of the executorch repo changes the value of
 # PYTORCH_NIGHTLY_VERSION, you should re-run this script to install the necessary
 # package versions.
-PYTORCH_NIGHTLY_VERSION=dev20240901
+PYTORCH_NIGHTLY_VERSION=dev20240925
 
 # Nightly version for torchvision
 VISION_NIGHTLY_VERSION=dev20240901
```

Comment:
we need to bump the vision version.  
atm I get this error running this PR:
ERROR: Cannot install torch==2.6.0.dev20240925 and torchvision==0.20.0.dev20240901+cu121 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested torch==2.6.0.dev20240925
    torchvision 0.20.0.dev20240901+cu121 depends on torch==2.5.0.dev20240901